src/components/AddItem/AddItem.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+
+export default props => {
+    const {bagName} = props;
+  return (
+    <>
+      <div className='p-1'>
+        <button
+          type="button"
+          className="btn btn-primary"
+          data-toggle="modal"
+          data-target="#exampleModalCenter"
+        >
+          Add Item
+        </button>
+
+        <div
+          className="modal fade"
+          id="exampleModalCenter"
+          tabIndex="-1"
+          role="dialog"
+          aria-labelledby="exampleModalCenterTitle"
+          aria-hidden="true"
+        >
+          <div className="col-10 modal-dialog modal-dialog-centered" role="document">
+            <div className="modal-content">
+              <div className="modal-header">
+                <h5 className="modal-title" id="exampleModalCenterTitle">
+                  Add Item To {bagName} Bag
+                </h5>
+                <button
+                  type="button"
+                  className="close"
+                  data-dismiss="modal"
+                  aria-label="Close"
+                >
+                  <span aria-hidden="true">&times;</span>
+                </button>
+              </div>
+              <div className="modal-body">...</div>
+              <div className="modal-footer">
+                <button
+                  type="button"
+                  className="btn btn-secondary"
+                  data-dismiss="modal"
```

Comment:
> `data-dismiss="modal"`

What is this `data-dismiss` for? Does some other component use it?

Reply:
Good catch, we're currently using bootstrap. Their modal logic uses the data-dismiss.